errors/now-next-legacy-mode.md

```diff
@@ -27,7 +27,7 @@ Serverless:
 - Runs `npm run now-build`
 - Does not run `npm install --production` as the output from the build is all that's needed to bundle lambdas.
 - No runtime dependencies, meaning smaller lambda functions
-- Optimized for fast [cold start](https://vercel.com/blog/serverless-ssr#cold-start)
+- Optimized for fast cold start
```

Comment:
We could possibly link to https://vercel.com/guides/how-can-i-improve-serverless-function-lambda-cold-start-performance-on-vercel, but that's a slightly different topic and I don't think it's worth doing at this point.